tests/FileReaderTest.php

```diff
@@ -78,4 +79,58 @@ public function testThrowsExceptionOnFail(): void
 
         $reader->read($nonExistentFilePath);
     }
+
+    private function createStdinStreamDouble(): object
+    {
+        return new class() {
```

Comment:
I know it was not there initially, but can we somehow add any typehint here other than `object` ?

Reply:
Unfortunately no, https://github.com/composer/composer/blob/2.6.6/src/Composer/Installer/BinaryInstaller.php#L248 does not have it either.